### PR TITLE
12202: Update 10203 Confirmation page

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
Navigate to https://staging.va.gov/education/apply-for-education-benefits/application/10203/confirmation

Select "Print this page"

There are two of the sentence under "What happens after I apply?". "You’ll get a Certificate of Eligibility (COE), or award letter, in the mail if we’ve approved your application. Bring this to the VA certifying official at your school." There should only be one.

[Originating Issue](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/12202)

## Testing done
Testing passes locally

## Screenshots
Regular View:
Screen Shot 2020-08-10 at 11 05 20 AM
Print View:
Screen Shot 2020-08-10 at 9 29 04 AM

## Acceptance criteria
- [x] Duplicate text is removed

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/applications/edu-benefits/10203/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -101,25 +101,20 @@ class ConfirmationPage extends React.Component {
...
-          <p className="confirmation-guidance-message">
+          <p>We usually decide on applications within 30 days.</p>
+
+          <p>
             You’ll get a Certificate of Eligibility (COE), or award letter, in
             the mail if we’ve approved your application. Bring this to the VA
             certifying official at your school.
           </p>
-          <p className="confirmation-guidance-message">
+          <p>
```
Comment (zurbergram) [inline src/applications/edu-benefits/10203/containers/ConfirmationPage.jsx:111]: what happened to the usages of `confirmation-guidance-message`?

Comment (elder-joseph) [reply src/applications/edu-benefits/10203/containers/ConfirmationPage.jsx:111]: I can add them back in if desired, but got an informal UX approval without them. Currently there is only one usage of that for this here:

```
<p className="confirmation-guidance-message">
            If you have questions, call 1-888-GI-BILL-1 (
            <a href="tel:+18884424551">1-888-442-4551</a>
            ), Monday &#8211; Friday, 8:00 a.m. &#8211; 7:00 p.m. ET.
</p>
```

Comment (zurbergram) [reply src/applications/edu-benefits/10203/containers/ConfirmationPage.jsx:111]: What did removing the css class do for the confirmation page?

Comment (elder-joseph) [reply src/applications/edu-benefits/10203/containers/ConfirmationPage.jsx:111]: Not much in the way of styling, it looks as though they may have been added as a way to select those components in a unit test as I see those are failing after being removed here.

**Review by zurbergram — APPROVED** [review]

**Review by sonntag-adam — APPROVED** [review]

**Pull request merged**